### DISCUSSION TITLE
Can drun show which desktop file it will execute?

### DISCUSSION THREAD

**Discussion by jasonish** [davatorium/rofi#1887] [Q&A]
I have a need to have the same application installed in a few different ways, for example, by the RPM and the Flatpak. Is it possible for the `drun` display to distinguish one from the other? Even if its just the full path to the .desktop file?

Thanks.

**Comment by DaveDavenport** [comment]
rofi follows the xdg standard for displaying desktop files and for what desktop file to  use and display:
https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html#desktop-file-id

> If multiple files have the same desktop file ID, the first one in the $XDG_DATA_DIRS precedence order is used.
